### PR TITLE
Why we are

### PR DESCRIPTION
**Pull request by bynikitachel**
TASK E-1, S-4, T-1

### CODE REVIEW

File: src/pages/home/sections/WhyWeAre/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  {
+    image: publishedWithChanges,
+    text: 'Есть вариант обмена/возврата',
+  },
+]
+
+const WhyWeAre = () => {
+  return (
+    <section className="why-we-are">
+      <div className="why-we-are-container">
```
Comment (imma-fish-u) [inline src/pages/home/sections/WhyWeAre/index.js]: why-we-are__container

File: src/pages/home/sections/WhyWeAre/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <HeaderSection headerText="Почему мы"></HeaderSection>
+        <div className="content-container">
+          {items && items.map((item, i) => {
```
Comment (imma-fish-u) [inline src/pages/home/sections/WhyWeAre/index.js]: why-we-are__content

**Review by imma-fish-u — CHANGES_REQUESTED** [review]

**Review by imma-fish-u — APPROVED** [review]

File: src/mockData/CarouselSliderData.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+export const CarouselSliderData = [
+    {
```
Comment (EgorAntonovich) [inline src/mockData/CarouselSliderData.js]: Is it really cool to contains links like hardcode?

File: src/mockData/NewProductData.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+export const NewProductData = [
+    {
```
Comment (EgorAntonovich) [inline src/mockData/NewProductData.js]: hardcode?

File: src/pages/home/index.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import OurWorks from './sections/OurWorks';
+import Reviews from './sections/Reviews';
+
+const Home = () => {
+    const descriptionTextWorks = 'Фото с комплектом "Кармен": в уютной квартире семьи Сергея и Татьяны не хватало акцентного элемента, который одновременно подчеркивал бы характер хозяев, а также украшал интерьер. В реализации идеи были использованы такие материалы как канвас и тюль сетка, а цвета подобраны под смелый запрос хозяев – красный и молочный.';
```
Comment (EgorAntonovich) [inline src/pages/home/index.js]: please contains const text in independent file

File: src/pages/home/index.scss [hunk 6]
```diff
@@ -0,0 +1,5 @@
+.main {
+    &__btn {
+        // @include button('basic', 'primary');
```
Comment (EgorAntonovich) [inline src/pages/home/index.scss]: why it is empty?

File: src/pages/home/sections/NewProducts/index.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import Gallery from '../../../../components/Gallery'
+
+const NewProducts = ({ galleryItems }) => {
+    return (
+      <Gallery
+        headerText="Новинки"
```
Comment (EgorAntonovich) [inline src/pages/home/sections/NewProducts/index.js:7]: are we really need to harcode this?

File: src/pages/home/sections/WhyWeAre/index.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import HeaderSection from '../../../../components/HeaderSection'
+import WhyWeAreItem from '../../../../components/WhyWeAreItem'
+
+const items = [
```
Comment (EgorAntonovich) [inline src/pages/home/sections/WhyWeAre/index.js]: What about contains const things in independent file?

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]

File: src/pages/home/sections/NewProducts/index.js (hunk 7 above)
Comment (bynikitachel) [reply src/pages/home/sections/NewProducts/index.js:7]: yes

**Pull request closed without merging**